tests/integration-tests/tests/common/utils.py

```diff
@@ -62,7 +62,7 @@
 }
 
 
-def retrieve_latest_ami(region, os, ami_type="official", architecture="x86_64", additional_filters=None):
+def retrieve_latest_ami(region, os, ami_type="official", architecture="x86_64", additional_filters=None, request=None):
```

Comment:
If ami_type is "pcluster", shall we set request as required parameter? Since there are other tests using retrieve_latest_ami when ami_type is "pcluster"

Reply:
How do you make it as required conditioned on the value of the other parameter?